tests/Admin/AdminTest.php

```diff
@@ -802,7 +802,7 @@ public function testGetPerPageOptions()
     {
         $admin = new PostAdmin('sonata.post.admin.post', 'NewsBundle\Entity\Post', 'SonataNewsBundle:PostAdmin');
 
-        $this->assertSame([16, 32, 64, 128, 192], $admin->getPerPageOptions());
+        $this->assertSame([16, 32, 64, 128, 256], $admin->getPerPageOptions());
```

Comment:
Please change the test so that you don't have to change them when checking the values. You could check that all the values are divisible by 4, for instance.